wordpress/versions/1.1.1/manifest/application.yaml.template

```diff
@@ -0,0 +1,89 @@
hJtyV4hFzUfrZ1nXlIOuMqRFy8fEyg3TFT4ujEzA1solYUKpwa/iUPMIzb0GgJ9KwM+EGu0nFs6jpK0hbFZUWUbMXDJfwTnuEd+qQbfuAtBT9TJ/tY1YnkTSmxiqHMKKq0knfVY2EZUC9iXIAxf8bl/Ve7zfKCAiEPlxS6dT/VK9XpaHymwCp+HnxQkJg+oZYSnA/V3VxCCQsE8EoIWxiEgTjBkyCWdi7t4hbYLc18wvlLFZdwTVkVKBx2BVj04gHKVXNwnLj1lJrHyfoQAWNvkvJiHOaUDRq8JUyeoovq2MAS1bsEUe/X9Rq0EIEYEKiIV3EZLQzIA3H0DuGv5NzFwuZSguvF2UFGbNqC/mLx14sYOqTwBgEMbWkowmC2CJKYTCshnRURrEuSL9GjPS7BqpmPnKTGBzVesuAOBqzFEyTSBrOaulktgLtfjDVKqTLJdMREWk2SGUOlgaAG97q1whfl1PV+R1wfLKgIi9x4VyrwziSJbSowrbBxqJNp7n87RJ7GaMG2Zp8EoyMSaddFiGzHFsBNQ8NFNsEytY96ndKxUiwr0fxwAPdhO7uwswIAvJ+5ZME34hPq6AIdzbYS68xibJNxyEcU4KGtHYgWBZrRo4s3u3zcZxs4K3kFZXQ7ZLRRB+KYvjLrObmFlrYbt+sxQ0ivLW/QB0YevQri/x2f1NMmMpOaxcVEtmfxO4S+ahH5YQ5ieNQPGofhyXANwnoFrTyPYgAt3KIzGBeq1yUS1bouWyPNQiiY0UOJeOPO/XocAf9zN0DOCxYg5jI034TICfajZOl1nYfZIgQTAADco7P5WszGA5nIFgAe+ZNnkFmmUip6FEiYZIE7TpYpU2isy/iOgCzEyWVKGjKDt30fcAeM9b2JKZvpQaZmVOWogTbL1qZrgViJPN674WxwLx+BJvNOhBZ37HlC20YDf0BblTnUJAooYI0SboYrUViHMc/6g+IS8Hcm4XeuNAvt3KG2KGfOTHc1NIJ72ZiNhceYAtDRq3imcmf0VgJyyJGDYpiquiwICkEJjYC10Eni15OF6mIWggi/l/MoLHkGo56aS1WBLprJLdkvhdwzc8eG5j6WvpBkPVekTaPjJrMhM1aayxCLcultrPaSPNJs7I49t65EOiIZx+Q9QHObQ90B/B28k667UcS4nWSJTXYonO1DXJfIiKliBfPWbCBzrgL94lHPFPiHKR244jaxRpRASqVHMqTRYZmUOEzNgQZo6WZWI7jXzI69JcvFE5Pohnw+ZQ67vvbCkHxYc/ST/2LbH5kaXjXKOkbhnbwDPBY6hkt7ZdIO8KtXcJJ32CsIlF2C9p05XLanh8ZcI6kydOohgcEXOhzg0nWucHewAd1umZxL5g1DRZoXknFdPLrGc/KGFZuF0c/SJxDelPNeg0kuz57yv0cFyUBswYwknfikHxgZLZmAkkfVG3iNzdin+ApQlV9WVZd9OUToDBCYH2OmN1V3R98hLI3RRK88wR6cSZxSyMCEuQb53+DkBLeb9daxisIrA0C6XLzyOc9OIFWnqWK0PSM5YR+2sVuJvgPFT5lyi90W86Q+iC490BSTqk304UQj/EgVYhzSXHdZGkcAlrNsQlo2jNdq7D2gPmKkd+qOKOBlfxuEethJZROh8s2h2BBaUtZYD/AGpfU7OeUF1wYqQ9JHaSqfvkO/r0WuiNONnMb4SV9NzO/E964O6r0GnX3VnglXaBWlamsWO/atZK2cwvBVefw23dJCuIAYcKH+incVzK2y+iodxo5062vKdgCV9F03BbDF9Bz+w0RflA0gtugNmDg8InNYxyi5TSfr68EYPHkOpjWEaJGxkQ1N/KMZp2vSDDuJzgk7sCH2T9FUo7uPJi4Q8cD65hy9ae0QD3yc1stb6GLufa4Y4JmlQlVbNcs2JDF5GHZuJ8wsDpQfCnBgD8OtpuAI7gCsvaAt42cdODpRdEuRnyshTi1CJORvA9zWVwwd62WoBdRzoBMJ+vWczgfZR0V7RUua2N2NFBckjl0fUMmJqPtd2M7T14CKSBb0g6kIKVcoU4YVtjttJa96UY4mRCeMlU8XCqFkJIlx52nNVoMr4lNHB+c3PNUaT8klpOnbWm1JJMaQ/X9GucYzNegmHFZahd54eaDIY0hAp7HVlmsqdgr7UGQ7H1JFYw83jajXcMLAV2nmrzHgz1QbeHN/ElpVCJTGvkuZpCT4ElD12WcYx/bN9R6CDG9rD7XJ13jxfh5+wQ6hYPqjWi/rPjGqskiFoFCoUesJVVkJUBA5/kzI7nW2yhR2gqiSu7d8xIeBulweTIw2UWdNPufmMacVgm/pxg5Wtu8RBypq0zI5Fkab5wD5Rk0OSOT3pkqN2gTlAY6/+QfdFBW52x+r+kD2yPhEIYMqERR01bD/BiwesZoiJvPG2DAiZUElEyr7Glwt4jG3eGrfur7eGKxcsldaTEo9YOkKlj0+/cFM2PXsyatMcbHRp9mnYYywhd1HlJN8OtGaUp4VNldKinhmRc/+m2OKAZ6zJY6wkY/BQnbV27v80e55y64zyRWc6sr/3xy4rIfFNJeHiJKT/y0cZNFz5fQmxS1UWxOxA+/Tqonmp0G+nXnlWxqKxCWlNsWG5h4ZwwWjvs7OXpLAEeXQY+/bnSEeMDg1hxJGxZNc3eI6WJux6i47Xtxw1wzJnr0cMCWU4wEVGVn9lDwMcb0lGi4+E3eITKUaPdgC4+yOBiqjc9mG3t9Zn/Ofd9BArwLXv9w3EEoGa8rv2cPbDnSa5Yl9WQJOvJzk3adfxhVQjeA/PvPahVN5WjR5OOQ992bs8ZYGjJikcbT94/deFiXd3FGYsLrm2sKBZOvHsE+Pbo8zf9E4LWHXv0C3RyE/VDa9ihVlvwvpvLuEFT/t7/PuA64vkBg4Y7O/m7k67Jzd8loMsLQ7s3U/0fxjPdOnoOePeVXm85t3Ea4uMzxKmNc2Av3/5DW/Vp3YhDvKr/AR0q1oK1ce+KAAAAAElFTkSuQmCC
```

Comment:
Nit, it'd be nice to envsubst the icon here to make this yaml template less bulky.

Looks like environment variable limits are very large for linux, smaller for windows but still large enough

Reply:
Acknowledged.
This is a lot easier in Helm and a bit of work to do without (presumably why it hasn't been done here yet). Omitting for now, but maybe a potential follow-up improvement. Some ideas:
* something like `RUN export ICON="$(base64 icon.png)"` in the deployer Dockerfile
* part of APP_PARAMETERS in the makefile